ports/zeldadoi/gameinfo.xml

```diff
@@ -0,0 +1,13 @@
+ <?xml version="1.0"?>
+<gameList>
+	<game>
+		<path>./Zelda Dungeons of Infinity.sh</path>
+		<name>The Legend of Zelda: Dungeons of Inifity</name>
+		<desc>The Legend of Zelda: Dungeons of Infinity is the result of four years of development in Gamemaker Studio from its creator Justin Bohemier. The game makes use of a number of A Link to the Past's original assets to really capture the same aesthetic, with new features and assets added to support the new procedurally-generated dungeon crawling.</desc>
+		<image>./zeldadoi/splash.png</image>
```

Comment:
Should use a `cover.png` and preferably a mixv1 style to remain consistent with other ports. Port is also missing `screenshot.png`. This `splash.png` file is used as a brief loading screen.

Reply:
![zelda-doi](https://github.com/user-attachments/assets/9c8acb66-f949-4bb6-92e6-9cedd4f4aee3)

ICYMI

Reply:
Thanks. Was wondering about this and just assumed splash could be used for both. I will break out into separate files.